src/doc/markdown/page92_customer_getting_started_with_simulator.md

```diff
@@ -0,0 +1,173 @@
+            nanopb:amd64 \
+            python3-protobuf:amd64 \
+            wget:amd64 \
+            curl:amd64 \
+            nodejs:amd64 \
+            webpack:amd64 \
+            npm:amd64
+```
+
+## Configure Using CMake
+
+The `jaiabot` software is configured using CMake which (by default) then generates Makefiles that the `make` tool uses to invoke the C++ compiler and linker.
+
+This process is summarized by:
+
+```
+# make a directory for the generated objects
+mkdir -p build/amd64
+cd build/amd64
+# configure the project
+cmake ../..
+# build it (using make by default)
+cmake --build .
+# return to root directory
+cd ../..
+```
+
+Webpack should compile with 55 errors in this step. Ignore them for now and proceed to the next step. These errors will be resolved in the Build step below. 
+
+## Install Goby, MOOS, and Jaia Code
+
+```
+# add mirror of packages.gobysoft.org to your apt sources
+echo "deb http://packages.jaia.tech/ubuntu/gobysoft/1.y/ `lsb_release -c -s`/" | sudo tee /etc/apt/sources.list.d/gobysoft_release.list
+# install the public key for packages.gobysoft.org
+sudo apt-key adv --recv-key --keyserver keyserver.ubuntu.com 19478082E2F8D3FE
+# add packages.jaia.tech to your apt sources
+echo "deb http://packages.jaia.tech/ubuntu/continuous/1.y/ `lsb_release -c -s`/" | sudo tee /etc/apt/sources.list.d/jaiabot_release.list
+# install the public key for packages.jaia.tech
+sudo apt-key adv --recv-key --keyserver keyserver.ubuntu.com 954A004CD5D8CF32
+sudo apt update
+# install the Goby and MOOS apps
+sudo apt install goby3-apps goby3-moos goby3-gui moos-ivp-apps moosdb10 libmoos-ivp
+# install the jaia code
+sudo apt install jaiabot-apps jaiabot-python jaiabot-config jaiabot-web
+# optional: compiled Arduino sketches and upload scripts
+sudo apt install jaiabot-arduino
+# optional: compiled documentation to /usr/share/doc/jaiabot/html
+sudo apt install jaiabot-doc
+# optional: Goby clang tool interface definitions (publish/subscribe API) to /usr/share/jaiabot/interfaces
+sudo apt install jaiabot-interfaces
+```
+
+## Build
+
+From the root of the JaiaBot repo, run the build script:
+
+```
+./build.sh
+```
+
+## Build the Web Interface
+
+There are three components comprising the Jaia Web Interface.  These are described in the following sections.
+
+### Command Control
+
+Command Control is the main map page, which allows the user to define new missions by clicking to define waypoints, etc.  This web app is written in JavaScript and uses the React and OpenLayers libraries.  Command Control must be built as follows:
+
+```
+cd src/web/command_control
+./build.sh
+cd ..
+```
+
+This will place the product into the `command_control/dist/client` directory.
+
+### Server
+
+In the jaiabot/src/web directory run:
+
+```
+pip install -r requirements.txt
+```
+
+In addition to the `flask` module, you need to have the python `dccl` module installed. Clone this repository outside of your local copy of the JaiaBot repo.
```

Comment:
We should review this because we should not need the customer to clone another repository to get this python package. it should be included in the requirements.txt if needed